<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_3</class>
 <widget class="QDialog" name="dlg_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>录入宿舍楼基本信息</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>32</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>宿舍楼号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>如：qy13（沁苑13栋）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>81</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>栋长姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>如：liuhuan</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>值班室电话</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>如：87541234</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>182</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>房间数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="singleStep">
    <number>200</number>
   </property>
   <property name="value">
    <number>144</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>231</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>床位数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>576</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>每床位年住宿费</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="value">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>dlg_3</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>dlg_3</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
